src/porepy/fracs/fractures_2d.py

```diff
@@ -295,6 +300,48 @@ def as_graph(self, split_intersections=True):
         else:
             return G
 
+    def connected_networks(self):
```

Comment:
I am not sure this is something we really need - you can easily do this from the user side. I am afraid of overloading on utility functions. So unless you can convince me otherwise, I do not want to include this.

Reply:
And yes, I am aware the same can probably be said about a lot of the other functions in this class.